js/src/Map.js

```diff
@@ -244,6 +277,14 @@ export class LeafletMapView extends utils.LeafletDOMWidgetView {
       this.create_panes();
       this.layer_views.update(this.model.get('layers'));
       this.control_views.update(this.model.get('controls'));
+
+      var layer_list = this.model.get('layers');
+      for (let i = 1; i < layer_list.length; i++) { // starting index is 1, since we are not taking into account the TileLayer at index 0
```

Comment:
Why are we making a special case for the first layer? It's probably not needed. 

And this line can be re-written `layer_list.forEach((layer) => {`

Reply:
The first layer if not mistaken corresponds to the default basemap in a Map (OpenStreetMap.Mapnik). Initially, we thought we would not add subitems to this layer but you are right it is not necessary or needed to fulfill this condition. We tried to replace the for loop by the forEach proposition.